fontforge/http.c

```diff
@@ -663,7 +663,7 @@ return( true );
 FILE *URLToTempFile(char *url,void *_lock) {
     FILE *ret;
 
-    if ( strncasecmp(url,"http://",7)==0 )
+    if ( strncasecmp(url,"http://",7)==0 || strncasecmp(url,"https://",8)==0 )
```

Comment:
@jtanx, we can't pull features without upsetting people, and nobody is under the illusion that FontForge is secure. Besides, it doesn't look like fixing the request code is that difficult if we really wanted to do it.